app.rb

```diff
@@ -19,12 +19,25 @@ class MakersBNB < Sinatra::Base
   end
 
   post '/spaces/new' do
-    Space.create(name: params[:name], description: params[:description], price: params[:price], checkin: params[:checkin], checkout: params[:checkout])
+    Space.create(name: params[:name], description: params[:description],\
```

Comment:
Would it increase readability to have just one param per line?

Reply:
Definitely cleaner.